tests/upgrade/customPassword_test.js

```diff
@@ -0,0 +1,79 @@
+const assert = require('assert');
+const { SERVICE_TYPE } = require('../helper/constants');
+
+Feature('PMM upgrade tests for custom password');
+const { dashboardPage } = inject();
+
+const clientDbServices = new DataTable(['serviceType', 'name', 'metric', 'annotationName', 'dashboard', 'upgrade_service']);
+
+clientDbServices.add([SERVICE_TYPE.MYSQL, 'ps-single', 'mysql_global_status_max_used_connections', 'annotation-for-mysql', dashboardPage.mysqlInstanceSummaryDashboard.url, 'mysql']);
+clientDbServices.add([SERVICE_TYPE.POSTGRESQL, 'pgsql_pgss_pmm', 'pg_stat_database_xact_rollback', 'annotation-for-postgres', dashboardPage.postgresqlInstanceSummaryDashboard.url, 'postgresql']);
+// clientDbServices.add([SERVICE_TYPE.MONGODB, 'rs101', 'mongodb_connections', 'annotation-for-mongo', dashboardPage.mongoDbInstanceSummaryDashboard.url, 'mongodb']);
```

Comment:
Is there a plan to add this at a later date? Might need todo and or ticket number if there's a plan for it.

Reply:
Uncomented it.